frontend/javascripts/types/api_flow_types.js

```diff
@@ -562,6 +562,7 @@ export type APIJob = {
   +boundingBox: ?string,
   +type: string,
   +state: string,
+  +manualState: string,
```

Comment:
```suggestion
  +state: "UNKNOWN" | "SUCCESS" | "PENDING" | "STARTED" | "FAILURE" | "MANUAL",
  +manualState: "SUCCESS" | "FAILURE" | null,
```

Having these precise types here in the job type would add quite a bit to type safety. I don't know whether merely adding these annotations here will make flow happy. It could very well be that it will complain about `adaptJobState` afterwards. In that case, these two types (plus a celery type probably) would need to be extracted like this:

```js
export APIJobCeleryState = ?;
export APIJobState = "UNKNOWN" | "SUCCESS" | "PENDING" | "STARTED" | "FAILURE" | "MANUAL";
export APIJobManualState = "SUCCESS" | "FAILURE" | null;
```

Then, the new signature would be:

```js
adaptJobState(command: string, celeryState: APIJobCeleryState, manualState: APIJobManualState): APIJobState
```